### PR TITLE
Decode & rocAL - Add new YUV formats supported by VCN and fix python test and jupyter notebook

### PR DESCRIPTION
**Pull request by rrawther**

### CODE REVIEW

File: rocAL/docs/examples/image_processing/decoder.py [hunk 1]
```diff
@@ -2,11 +2,12 @@
 import sys
 from tkinter import W
```
Comment (LakshmiKumar23) [inline rocAL/docs/examples/image_processing/decoder.py:3]: @rrawther Why do we need tkinter? Can you remove it?

Comment (rrawther) [reply rocAL/docs/examples/image_processing/decoder.py:3]: Done

**Comment by paveltc** [comment]
@rrawther @LakshmiKumar23 @kiritigowda I ran decoder.py and it displayed 4 images for me that I could move around and manipulate with the GUI. Is there anything else that needs to be tested here?

**Comment by LakshmiKumar23** [comment]
@rrawther I can see the 4 images too. Not sure how you moved it around @paveltc.

**Comment by LakshmiKumar23** [comment]
@kiritigowda can be merged

**Review by LakshmiKumar23 — APPROVED** [review]

**Pull request merged**